### ISSUE TITLE
merge binlog events implementation frrom https://github.com/nevill/zongji

### ISSUE THREAD

**Issue by sidorares** [sidorares/node-mysql2#148]
zongji client seems to be more advanced than current binlog parser in node-mysql2

also would be interesting to see node-mysql2 working with https://github.com/numtel/mysql-live-select

**Comment by zenflow** [comment]
This would be awesome. I might have to make a go at a PR when I have time.

@sidorares What do you think of adding live-select capabilities directly to this package?

**Comment by sidorares** [comment]
> What do you think of adding live-select capabilities directly to this package?

not %100 sure but possible. I'd probably start with exposing all required events from binlog and maybe later add api to make it simple to use them

**Comment by sidorares** [comment]
@zenflow note that @shtse8 already started working on this in #685

**Comment by shtse8** [comment]
Yes, surely support adding this to make MySQL better for live event with binlog. Live data is common nowadays but not many tools supporting working with MySQL.

**Comment by Fabbok1x** [comment]
Hey guys, seeing some new releases. Has any of this been addressed in the new releases and maybe forgotten in here?

**Comment by sidorares** [comment]
@Fabbok1x don't think anything happened on this front. Happy to try to reignite
